### PR TITLE
Remove mentions of Lightbend support, switch to Plausible analytics

### PR DESCRIPTION
**Pull request by julienrf**
image

image

### CODE REVIEW

**Comment by julienrf** [comment]
I noticed that the API documentation was not built. I guess it is produced separately?

File: src/landing/_template/error.st [hunk 1]
```diff
@@ -12,15 +12,7 @@
...
-
-    <!-- OneTrust Cookies Consent Notice (Production Standard, scala-sbt.org, en-GB) start -->
-    <script src="https://optanon.blob.core.windows.net/consent/d759c5db-6821-46e0-a988-6bc699efb74e.js" type="text/javascript" charset="UTF-8"></script>
-    <script type="text/javascript">
-      function OptanonWrapper() {
-        //callback when OneTrust is loaded and ready
-      }
-    </script>
-    <!-- OneTrust Cookies Consent Notice (Production Standard, scala-sbt.org, en-GB) end -->
+    <script defer data-domain="scala-sbt.org" src="https://plausible.scala-lang.org/js/script.js"></script>
```
Comment (eed3si9n) [inline src/landing/_template/error.st:15]: Could you explain what this does please?

**Comment by eed3si9n** [comment]
> I noticed that the API documentation was not built. I guess it is produced separately?

Yes, it's built using sbt/sbt's special branch and sbt-unidoc because we have multiple repos.

File: src/landing/_template/error.st (hunk 1 above)
Comment (julienrf) [reply src/landing/_template/error.st:15]: It removes the banner “This site uses cookies to offer you a better browsing experience…”, and loads the Plausible analytics scripts hosted on the scala-lang.org servers. Plausible is an analytics platform that does not use cookies. Maybe cookies are used by another component of the website?

**Review by eed3si9n — APPROVED** [review]
LGTM pending CI

**Pull request merged**